gst/mfx/gstmfxpostproc.c

```diff
@@ -851,7 +855,9 @@ gst_mfxpostproc_transform_caps_impl (GstBaseTransform * trans,
     GstPadDirection direction, GstCaps * caps)
 {
   GstMfxPostproc *const vpp = GST_MFXPOSTPROC (trans);
+#ifdef HAVE_GST_GL_LIBS
```

Comment:
Not sure if this macro check is required.

Reply:
Definitely flagged as warning. We need this.